src/main/flight/pid.c

```diff
@@ -445,19 +446,63 @@ STATIC_UNIT_TESTED FAST_CODE_NOINLINE float pidLevel(int axis, const pidProfile_
 #ifdef USE_GPS_RESCUE
     angle += gpsRescueAngle[axis] / 100; // ANGLE IS IN CENTIDEGREES
 #endif
-    angle = constrainf(angle, -levelAngleLimit, levelAngleLimit);
     const float errorAngle = angle - ((attitude.raw[axis] - angleTrim->raw[axis]) / 10.0f);
+    float angleErrorCorrection = errorAngle * pidRuntime.levelGain;
+    const float levelErrorGainBoost = (1.0f + (fabsf(errorAngle) / levelAngleLimit)); //required here for debug later
```

Comment:
Ffrom quite a bit of testing, boosting gain on faster moves (ie with more error) can lead to P wobble during the move if the error gain is already close to oscillation.  This is then amplified in the PID loop and isn't helpful.

I think we should remove the gain boost completely.

Reply:
Agreed, P boost shouldn't be necessary with feedforward. Removed.